cmd/image-builder/README.md

```diff
@@ -1,290 +1,175 @@
 # Image Builder
 
-Image Builder is a tool for building OCI-compliant images.
-It can build images using different backends, such as Kaniko, BuildKit, and Azure DevOps (ADO).
-It also supports signing images with a pre-defined set of signing services
-to verify that the image comes from a trusted repository and has not been altered in the meantime.
-The tool is designed to be used in ProwJobs.
+Image Builder is a tool for building OCI-compliant images in SLC-29 compliant system from GitHub workflow.
+It signs images with a signify service to verify that the image comes from a trusted repository and has not been altered in the meantime.
```

Comment:
Do you mean `signing service` (as in the deprecated doc)? If you do, please also change the other occurrences of `signify`.

Reply:
No, the `signify` is the correct name. This is the name of signing service we are using.